platform/ios/.gitignore

```diff
@@ -0,0 +1,2 @@
+ios.xcodeproj/xcuserdata
+ios.xcworkspace/xcuserdata
```

Comment:
No `xcuserdata` should ever be committed in my opinion. The shared settings live alongside these directories, in `xcshareddata`.